wafer/talks/templates/wafer.talks/speakers.html

```diff
@@ -0,0 +1,20 @@
+{% extends "wafer/base.html" %}
+{% load i18n %}
+{% block content %}
+    <div class="container">
+            {% for user_profile in speakers %}
+                {% if forloop.counter0|divisibleby:"4" %}
+                    <div class="row">
+                {% endif %}
+                    <div class="col-md-3">
+                        <img class="thumbnail center-block" src="{{ user_profile.avatar_url }}">
+                        <h3 class="text-center">
+                            {{ user_profile.user.get_full_name|default:user_profile.user.username }}
+                        </h3>
+                    </div>
+                {% if forloop.counter|divisibleby:"4" %}
+                    </div>
+                {% endif %}
```

Comment:
Isn't this going to leave us missing a </div> if the number of speakers isn't divisible by 4?